packages/react/src/Card/Card.module.css

```diff
@@ -0,0 +1,29 @@
+.Card {
+  max-width: var(--brand-Card-maxWidth);
+  display: flex;
+  flex-direction: column;
+  text-decoration: none;
+  color: var(--brand-color-text-default);
+  background-color: var(--brand-Card-background-default);
+  border: 1px solid var(--brand-color-border-default);
+  box-shadow: var(--brand-Card-visual-shadow);
+  border-radius: 6px;
+  padding: var(--base-size-32);
+  transition: transform 0.6s cubic-bezier(0.16, 1, 0.3, 1);
```

Comment:
Stole this transition + hover from the `resourcesCard` component in .com pages. Not sure if these need to be a custom Card tokens 🤔

Reply:
I think it should be, yeah 👍. We have a similar token for `--brand-control-animation-easing`, which is possibly even the same value.

Reply:
It is close, but not exactly the same. We use the `brand control animation` tokens for small elements like checkboxes show/hide I think for this case `Cards` deserve an special animation treatment as it's a "bigger" element on the interface.